### PR TITLE
Update ca from 1.60 to 1.73 + Add debug info on ca repository

### PR DESCRIPTION
**Pull request by guillaume-philippon**
Every thing is on title

### CODE REVIEW

File: security/ca-policy-egi-core.pan [hunk 1]
```diff
@@ -1,115 +1,114 @@
 #
-# @(#)ca-policy-egi-core.tpl 1.60-1
-# Copyright (c) Stichting European Grid Initiative, 2014
+# @(#)ca-policy-egi-core.tpl 1.73-1
+#
+# Copyright (c) Stichting EGI, 2016
+#
 # This work is licensed under the Creative Commons Attribution-Noncommercial
-# 3.0 License. To view a copy of this license, visit
+# 3.0 License. To view a copy of this license, visit
```
Comment (wdpypere) [inline security/ca-policy-egi-core.pan]: remove trailing space.

File: repository/config/ca.pan [hunk 2]
```diff
@@ -4,7 +4,7 @@ variable YUM_SNAPSHOT_NS ?= 'repository/snapshot';
 
 @{
 desc =  namespace of template associated with CA RPM YUM snapshot
-values =  string
+values =  string
```
Comment (guillaume-philippon) [inline repository/config/ca.pan]: done with --amend option

File: security/ca-policy-egi-core.pan (hunk 1 above)
Comment (guillaume-philippon) [inline security/ca-policy-egi-core.pan]: done with --amend option

File: repository/config/ca.pan [hunk 3]
```diff
@@ -18,10 +18,13 @@ default = YUM_SNAPSHOT_DATE
 required = no
 }
 variable YUM_CA_RPM_SNAPSHOT_DATE ?= if ( is_null(YUM_CA_RPM_SNAPSHOT_DATE) ) {
```
Comment (ned21) [inline repository/config/ca.pan]: I don't follow this code. If YUM_CA_RPM_SNAPSHOT_DATE is not null or undef then this code will not run because of the ?= But if it is_null then you return SELF which will set it to undef. The addition of debug() statements seem to be a plaster over a block that should just be re-written to be less convoluted?

File: repository/config/ca.pan [hunk 4]
```diff
@@ -37,8 +40,10 @@ include { 'quattor/functions/repository' };
 
 '/software/repositories' = {
   if ( is_defined(YUM_CA_RPM_SNAPSHOT_DATE) && (QUATTOR_RELEASE >= '14') ) {
+    debug(format('%s: adding CA repository for YUM snapshot %s',OBJECT,YUM_CA_RPM_SNAPSHOT_DATE));
     add_repositories(SECURITY_CA_RPM_REPOSITORY,YUM_CA_RPM_SNAPSHOT_NS);
```
Comment (ned21) [inline repository/config/ca.pan]: This checks and logs YUM_CA_RPM_SNAPSHOT_DATE but then adds YUM_CA_RPM_SNAPSHOT_NS to the repositories tree. Why not check and log YUM_CA_RPM_SNAPSHOT_NS instead?

File: security/ca-policy-egi-core.pan [hunk 5]
```diff
@@ -1,115 +1,114 @@
...
-# Copyright (c) Stichting European Grid Initiative, 2014
+# @(#)ca-policy-egi-core.tpl 1.73-1
+#
+# Copyright (c) Stichting EGI, 2016
+#
 # This work is licensed under the Creative Commons Attribution-Noncommercial
 # 3.0 License. To view a copy of this license, visit
 #   http://creativecommons.org/licenses/by-nc/3.0/
-# Generated by ./builddist-egi.pl
+# Generated by ./builddist-egi-multi.pl
```
Comment (ned21) [inline security/ca-policy-egi-core.pan]: Changes in this file are OK. If the addition of debug is controversial/needs discussion then perhaps it should be a separate commit to allow it to proceed without being blocked by the questions above?

File: repository/config/ca.pan (hunk 3 above)
Comment (guillaume-philippon) [inline repository/config/ca.pan]: I just add some debug info. I will have a deep look in the code itself.

File: repository/config/ca.pan (hunk 4 above)
Comment (guillaume-philippon) [inline repository/config/ca.pan]: YUM_CA_RPM_SNAPSHOT_NS is always defined. We must add this specific repository only if you use a QUATTOR_RELEASE >= '14' and SNAPSHOT_DATE is defined.

**Comment by jouvin** [comment]
@guillaume-philippon please could you rebase your PR so that we can include it in 16.10.

**Comment by guillaume-philippon** [comment]
I merged with upstream
